SDWebImage/FLAnimatedImage/FLAnimatedImageView+WebCache.m

```diff
@@ -119,7 +132,7 @@ - (void)sd_setImageWithURL:(nullable NSURL *)url
                     placeholderImage:placeholder
                              options:options
                         operationKey:nil
-                       setImageBlock:^(UIImage *image, NSData *imageData) {
+               internalSetImageBlock:^(UIImage * _Nullable image, NSData * _Nullable imageData, SDImageCacheType cacheType, NSURL * _Nullable imageURL) {
```

Comment:
Since this is `internalSetImageBlock`, and we also associated the image url, is it necessary for this `imageURL `  parameter to exist? Or is there some other place use for this parameter ?

Reply:
Thanks, I get it!

Reply:
Any extra idea ? This comment seems about the `imageUrl` arg, but it's not related to the PR's changes...

Reply:
Emm, this `PR` has merged, if we make it as an internal method, we don't need to add this `imageURL`, may not have any benefit.